src/middleware/parser.js

```diff
@@ -55,17 +55,26 @@ const parseIncomingBody = async (ctx, inputFormat, next) => {
 }
 
 exports.parseBodyMiddleware = metaData => async (ctx, next) => {
+  const incomingContentType = ctx.get('Content-Type').split('/')[1]
   try {
     // parse incoming body
-    await parseIncomingBody(ctx, metaData.transformation.input, next)
+    await parseIncomingBody(ctx, incomingContentType.toUpperCase(), next)
 
     // wait for middleware to bubble up before running the below method
 
     // parse outgoing body
-    parseOutgoingBody(ctx, metaData.transformation.output)
+    parseOutgoingBody(ctx, metaData.transformation.output.toUpperCase())
```

Comment:
I dont think we need to change this variable to be uppercase as it should already be defined as uppercase. I think this is prob from something i introduced but having a look at it a bit more, its a bit confusing with all the toUpperCase/toLowerCase methods. 

Wondering, should we allow a specific format or allow both upper and lowercase?

Reply:
@BMartinos Yeah, I think we should convert it to one case (probably lower) before we start processing. That way, the input could be in upper or lowercase, but it only ever is in one state in the program. Currently, the headers convert to all lowercase at some point before this code, which means that if we only check for uppercase, at some places the checks don't match.

Reply:
ok cool im happy with that. I think the issue i was facing initially is that technically seeing as they are acronyms, they should be captured and displayed in uppercase. but the content-type headers are in lowercase. Im happy to have them converted to lowercase and use as such everywhere for now.